src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/StatistikkUtils.java

```diff
@@ -0,0 +1,22 @@
+package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import org.jetbrains.annotations.NotNull;
+
+public class StatistikkUtils {
+
+    /**
+     * Source of trouth for kalkulering av sykefraværsprosent basert på tapte dagsverk og mulige
+     * dagsverk. ArithmeticException kastes dersom muligeDagsverk er 0.
+     */
+    public static BigDecimal kalkulerSykefraværsprosent(
```

Comment:
Likte både tanken av "source of trouth" for noe veldig sentralt og navn til metoden. Lett å finne frem og å forstå 💯